src/App.php

```diff
@@ -617,7 +618,7 @@ protected function createRequestPathFromLocalPath(string $localPath): string
             } else {
                 $request = \Symfony\Component\HttpFoundation\Request::createFromGlobals();
```

Comment:
TODO for myself - this relies on $_GET, rework it in https://github.com/atk4/ui/pull/2095 and test by unsetting `$_GET`